pom.xml

```diff
@@ -3,9 +3,9 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>cloud.solarwindscloud</groupId>
+  <groupId>cloud.solarwinds</groupId>
```

Comment:
Can we do `com` instead of `cloud` because `cloud` doesn't look like a real TLD?

Reply:
Done